.github/actions/feedstock-optional-pr/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Open-CE Feedstock PR Optional Tests
+
+author: "Open-CE"
+
+description: "Optional global PR tests for Open-CE feedstocks."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install dependencies
+      shell: bash -l {0}
+      run: |
+        conda install -y conda-build
+    - name: Run PR Tests
+      shell: bash -l {0}
+      run: |
+        test_path="$GITHUB_ACTION_PATH/../../../tests"
+        python $test_path/feedstock_tests/feedstock-optional-pr.py
```

Comment:
I think this needs changed to `check_build_numbers.py` ?

Reply:
@jayfurmanek 
You're right. Thanks for catching this. It's fixed now.